stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -1003,6 +1032,24 @@ public class MessageListViewModel(
         }.exhaustive.enqueue()
     }
 
+    /**
+     * Scrolls to message if in list otherwise get the message from backend.
+     *
+     * @param message The message we wish to scroll to.
+     */
+    public fun scrollToSelectedMessage(message: Message) {
```

Comment:
To me this seems like an entry point into focusing a message after it's loaded. Instead of having a new parameter and property for the scroll, we can simply pass the required message to `loadMessage` and then in its `enqueue` wait for the positive result and attempt to focus